weblate/vcs/git.py

```diff
@@ -279,6 +280,20 @@ def configure_remote(self, pull_url, push_url, branch):
         )
         self.branch = branch
 
+    def api_url(self):
+        slug = self.component.repo.split("/")[-1].replace(".git", "")
+        owner = self.component.repo.split("/")[-2]
+        return "https://api.github.com/repos/{0}/{1}".format(owner, slug)
```

Comment:
Please move this to GitHub class.

Reply:
Aah. Makes sense. This is very specific to GitHub.